apps/test/unit-tests.js

```diff
@@ -1,7 +1,19 @@
+// This gets replaced by karma webpack with the updated files on rebuild
```

Comment:
nit: I think this comment belongs to `__karmaWebpackManifest__ ` not `babel-polyfill`